src/components/AuthContainer.js

```diff
@@ -16,7 +16,7 @@ import TermsOfService from 'src/pages/TermsOfService'
 const AuthContainer = ({ children }) => {
   const { name, public: isPublic } = useRoute()
   const { isSignedIn, registrationStatus, termsOfService, profile } = useStore(authStore)
-  const userNeedsToAcceptTos = termsOfService.userNeedsToAcceptTos
+  const displayTosPage = isSignedIn && (!!!termsOfService.userContinuedUnderGracePeriod || !termsOfService.userCanUseTerra)
```

Comment:
I'm assuming `!!` is used to cast as a boolean and then `!` to negate that? If so, one `!` is sufficient and more readable.

```suggestion
  const displayTosPage = isSignedIn && (!termsOfService.userContinuedUnderGracePeriod || !termsOfService.userCanUseTerra)
```